doc/03_research/04_requirements/03_requirements.md

```diff
@@ -0,0 +1,82 @@
+# Requirements
+
+**Summary:** This page contains the requirements obtained from the Carla Leaderboard website as well as former projects in the `Praktikum Autonomes Fahren`
+
+---
+
+## Author
+
+Josef Kircher, Simon Erlbacher
+
+## Date
+
+17.11.2022
+
+## Prerequisite
+
+---
+<!-- TOC -->
+* [Requirements](#requirements)
+  * [Author](#author)
+  * [Date](#date)
+  * [Prerequisite](#prerequisite)
+  * [Requirements from Leaderboard tasks](#requirements-from-leaderboard-tasks)
+  * [Carla Leaderboard Score](#carla-leaderboard-score)
+  * [Prioritized driving aspects](#prioritized-driving-aspects)
+  * [more Content](#more-content)
+    * [Sources](#sources)
+<!-- TOC -->
+
+## Requirements from Leaderboard tasks
+
+* follow waypoints on a route
+* don't deviate from route by more than 30 meters
+* act in accordance with traffic rules
+* don't get blocked
+* complete 10 routes (2 weather conditions)
+
+---
+
+## Prioritized driving aspects
+
+There are different ways to prioritize the driving aspects mentioned in the document [08_use_cases](https://github.com/ll7/paf22/blob/482c1f5a201b52276d7b77cf402009bd99c93317/doc/03_research/08_use_cases.md).
+The most important topics, in relation to this project, are the driving score and the safety aspect.
+Also, it is appropriate to implement the basic features of an autonomous car first. The list is a mixture of the different approaches. Prioritizing from very important functionalities to less important features.
+
+`Very important:`
+
+* Recognize the street limitations

```

Comment:
Maybe "hold line" is also a very important task. Or is it included in "steering, accelerate, decelerate"?